core/scrollbar.js

```diff
@@ -992,14 +1062,16 @@ Blockly.Scrollbar.prototype.updateMetrics_ = function() {
 
 /**
  * Set the scrollbar handle's position.
- * @param {number} value The distance from the top/left end of the bar, in CSS
- *    pixels.  It may be larger than the maximum allowable position of the
- *    scrollbar handle.
+ * @param {number} value TBD
  * @param {boolean=} updateMetrics Whether to update metrics on this set call.
  *    Defaults to true.
  */
 Blockly.Scrollbar.prototype.set = function(value, updateMetrics) {
-  this.setHandlePosition(this.constrainHandle_(value * this.ratio));
+  var maxScrollDistance = this.horizontal_ ?
+      this.oldHostMetrics_.contentWidth - this.oldHostMetrics_.viewWidth :
+      this.oldHostMetrics_.contentHeight - this.oldHostMetrics_.viewHeight;
+  var ratio = (this.scrollViewSize_ - this.handleLength_) / maxScrollDistance;
```

Comment:
Ok, so what I think this is is the "Amount to move the scrollbar for every pixel that the workspace is moved."
I think that should at least go into a comment above this value, but I am not entirely sure what that translates to for a name just yet haha.

Reply:
I've created a class property with jsdoc description and moved the computation to `resizeContentVertical`/`resizeContentHorizontal` (with some added variable names/comments to hopefully help with understanding).